db/models/meme.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const {STRING, DECIMAL, TEXT, INTEGER} = require('sequelize')
+
+module.exports = db => db.define('memes', {
+  name: {
+  	type: STRING,
+  	allowNull: false,
+  },
+  price: {
+  	type: DECIMAL(10,2),
+  	defaultValue: 100.00
+  },
+  rating: {
+  	type: DECIMAL(10,2),
+  }, 
+  photo: {
+  	type: STRING,
+  	defaultValue: '',
+  },
+  product_info: {
+  	type: TEXT,
+  	defaultValue: '',
```

Comment:
Perhaps rather than having an empty string value, set default to null? Null is more expressive.  Right now this allows the person uploading product info to upload with empty text, this could lead to bugs. We probably want a required text description.